packaging/Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM registry.opensource.zalan.do/library/alpine-3.13:latest
+ARG BASE_IMAGE=registry.opensource.zalan.do/library/alpine-3.13:latest
```

Comment:
Let's keep the public single-arch approved base image as default for now since this project is open source.